compiler/src/dmd/backend/mscoffobj.d

```diff
@@ -1176,7 +1176,7 @@ void MsCoffObj_ehtables(Symbol* sfunc,uint size,Symbol* ehsym)
 @trusted
 private void emitSectionBrace(const(char)* segname, const(char)* symname, int attr, bool coffZeroBytes)
 {
-    char[16] name = void;
+    char[16] name;
```

Comment:
Subsequent lines do not check for overflowing name[]. Perhaps refactor this code to use D arrays rather than C pointers?